policyengine_canada/parameters/gov/provinces/bc/tax/income/credits/bc_age_credit/base.yaml

```diff
@@ -0,0 +1,12 @@
+description: British Columbia age credit base amount.
```

Comment:
could you rename the folder `age`

Reply:
Should I rename the entire "bc_age_credit" folder or just one specific file?

Reply:
folder. in general we don't need parameter folders to restate other parts of folder tree

Reply:
Ok thank you! I have also renamed the tax reduction folder.